src/components/HomePageHeader/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext'
+import GitHubButton from 'react-github-btn'
+
+import css from './styles.module.css'
+
+function HomepageHeader() {
+  const { siteConfig } = useDocusaurusContext()
+
+  return (
+    <header className={css.hero}>
```

Comment:
As feedback: I removed clsx because I saw this due those lines:

![image](https://user-images.githubusercontent.com/11404065/213267316-64677dd7-b572-4463-8711-73207c198b28.png)